app/src/main/java/com/example/timer/DataAdapter.java

```diff
@@ -182,15 +190,36 @@ public boolean onLongClick(View view) {
         }
     }
 
-    interface OnTimerListener {
-        void onTimerClick(int position);
+    private void showIcons(PopupMenu popupMenu){
+        try {
+            Field[] fields = popupMenu.getClass().getDeclaredFields();
+            for (Field field : fields) {
+                if ("mPopup".equals(field.getName())) {
+                    field.setAccessible(true);
+                    Object menuPopupHelper = field.get(popupMenu);
+                    Class<?> classPopupHelper = Class.forName(menuPopupHelper.getClass().getName());
+                    Method setForceIcons = classPopupHelper.getMethod("setForceShowIcon",
+                            boolean.class);
+                    setForceIcons.invoke(menuPopupHelper, true);
+                    break;
+                }
```

Comment:
What's that interesting piece of code?)

Reply:
This part of code is responsable for displaying icons in menu. I showed you this function about a months ago, and you said that's ok. Unfortunately, i didn't find any other solution.

Reply:
###  I mean this icons in popup menu for each timer.
![8Clx3nlXS38](https://user-images.githubusercontent.com/62645506/100892194-1e857400-34cb-11eb-927a-1ece27bdb0d5.jpg)